WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -92,6 +92,8 @@ final class ProductVariationsViewController: UIViewController {
 
     private var product: Product {
         didSet {
+            configureRightButtonItem()
+            configureEmptyState()
```

Comment:
nit: usually we use `configure*` for one-time setup, and `update*` for setup that could change so that it's easier to tell if a setup stays the same 🙂

Reply:
Nice, I wasn't aware of that convention!